Draft code of conduct

bast opened the pull request:
Origin of this:

> This code of conduct is copied and adapted from the code of conduct used at the
> [Collaborations Workshop 2020](https://www.software.ac.uk/cw20/code-conduct)
> (CC-BY-NC 2.5 license) which was adapted from the example policy at the
> [Geek Feminism wiki](https://geekfeminism.wikia.org/wiki/Conference_anti-harassment/Policy),
> created by the Ada Initiative and other volunteers (CC-0 license).
> The procedure for reporting harassment has been adopted from the Ada Initiative's guide titled
> ["workshop anti-harassment/Responding to Reports"](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Responding_to_reports).

We need to modify "Here we will list members of the code of conduct committee". Set as draft PR until we modify this.

Can you please have a look @annefou @cosden?

cosden commented on code-of-conduct.md:
```diff
@@ -0,0 +1,93 @@
+
+If a participant engages in behaviour that violates this code of conduct, the
+workshop organisers may take any action they deem appropriate, including
+warning the offender or expulsion from the workshop with no refund.
```
Unless there is something I'm not thinking of, it's not going to cost anyone anything so I'd suggest removing "with no refund."

bast replied:
Thanks. This is a copy-paste error. I will fix.

cosden commented on code-of-conduct.md:
```diff
@@ -0,0 +1,93 @@
+If you are being harassed, notice that someone else is being harassed, or have
+any other concerns, please contact any member of the code of conduct committee:
+
+**Here we will list members of the code of conduct committee.**
```
I assume email addresses would be listed? Is that sufficient, or should we suggest another method of contact?

bast replied:
Email addresses or direct message on RSE Slack?

sdruskat replied:
I think we said, based on @SimonHettrick's suggestion, that we should list one email address, that of the chair. So one action point for the committee would be to decide on who will chair, and add their email address.

sdruskat replied:
DMs on Slack are also fine I think.

bast replied:
I agree, as participant I would like to see several contact points so that nobody is outside of the system.

bast replied:
Addressed this but did not add email addresses of others but I will summarize remaining steps in a moment.

cosden commented on code-of-conduct.md:
```diff
@@ -0,0 +1,93 @@
+and how to further proceed with the situation.  All of our committee members
+are informed of the code of conduct policy and guide for handling harassment at
+the workshop. There will be a mandatory code of conduct committee meeting just
+prior to the event when this will be reiterated as well.
```
The final paragraph is the only thing that I think we need to discuss, agree on, and possibly publish. Because as it stands now, I don't know exactly how we'd handle it. 
It's not clear from the language, but who does the mandatory code of conduct meeting apply to?

sdruskat replied:
I think we could include a review of the code by the committee in one of the next calls, once the procedure of handling instances is clear. IMHO this would suffice to keep this point in the text and publish it.

cosden replied:
Sounds good to me. I think it's sufficient to have a discussion on an upcoming call.

bast replied:
I tried to make this clearer based on this good feedback.

cosden commented:
Overall I think this looks good! The only thing that _might_ be missing is something about what will happen after a report. I'm not convinced its necessary, but we could say something like: A decision will be made as quickly as possible and any decisions made by the code of conduct will be enforced immediately.

bast commented:
I have removed the "draft" label and hopefully addressed the feedback.

At least one thing remains to be adjusted. If Anne and Ian are comfortable having their email addresses here, we should add them. Alternatively other forms of contacting:
```
If you are being harassed, notice that someone else is being harassed, or have
any other concerns, please contact any member of the code of conduct committee
via email or direct message using the workshop Slack chat:

- Anne Fouilloux
- Ian Cosden
- Radovan Bast (radovan.bast@uit.no)
```

annefou commented:
?I am happy to add my email address (annefou@uio.no)


anne.

________________________________
From: Radovan Bast <notifications@github.com>
Sent: 07 September 2020 14:21
To: RSE-leaders/researchsoftware.org
Cc: Anne Claire Fouilloux; Mention
Subject: Re: [RSE-leaders/researchsoftware.org] Draft code of conduct (#19)


I have removed the "draft" label and hopefully addressed the feedback.

At least one thing remains to be adjusted. If Anne and Ian are comfortable having their email addresses here, we should add them. Alternatively other forms of contacting:

If you are being harassed, notice that someone else is being harassed, or have
any other concerns, please contact any member of the code of conduct committee
via email or direct message using the workshop Slack chat:

- Anne Fouilloux
- Ian Cosden
- Radovan Bast (radovan.bast@uit.no)


-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/RSE-leaders/researchsoftware.org/pull/19#issuecomment-688289387>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AB6KIPDKHXWABAZOMYKDEHDSETF3BANCNFSM4PICPTBQ>.

danielskatz commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+
+The important information we need consists of:
+- Identifying information (name) of the participant doing the harassing
+- The behaviour that was in violation
```
```suggestion
- The behaviour that was potentially in violation
```

danielskatz commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+
+The code of conduct committee is well informed on how to deal with the incident
+and how to further proceed with the situation.  All of our committee members
+are informed of the code of conduct policy and guide for handling harassment at
```
```suggestion
are knowledgable about the code of conduct policy and how to handle harassment at
```

danielskatz commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+as quickly as possible and any decisions made by the code of conduct will be
+enforced immediately.
+
+The code of conduct committee is well informed on how to deal with the incident
```
```suggestion
The code of conduct committee is well informed on how to deal with an incident
```

danielskatz replied:
```suggestion
The code of conduct committee is well informed on how to deal with the incident.
```

danielskatz commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+enforced immediately.
+
+The code of conduct committee is well informed on how to deal with the incident
+and how to further proceed with the situation.  All of our committee members
```
```suggestion
All of our committee members
```

danielskatz commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+as quickly as possible and any decisions made by the code of conduct will be
+enforced immediately.
+
+The code of conduct committee is well informed on how to deal with the incident
```
```suggestion
The code of conduct committee is well informed on how to deal with an incident.
```

cosden commented on code-of-conduct.md:
```diff
@@ -0,0 +1,100 @@
+via email or direct message using the workshop Slack chat:
+
+- Anne Fouilloux (annefou@uio.no)
+- Ian Cosden
```
```suggestion
- Ian Cosden (icosden@princeton.edu)
```

bast commented:
Hopefully everything addressed. I think this is ready to go.

sdruskat commented:
:tada: Thanks @radovan, @annefou, @cosden  and @danielskatz for your work on this! I'll move this to the `/2020-workshop/` folder which already holds the agenda, and merge.
